packages/UI/src/context/UserDataContext.js

```diff
@@ -2,13 +2,35 @@ import constate from 'constate'; // State Context Object Creator
 import { useEffect, useState } from 'react';
 import axios from "axios";
 
+// export const DEFAULT_USER = {
```

Comment:
We need to keep the default user as is, otherwise users will see Kris' info when they are not logged in which will cause a lot of confusion.